applications/resturant/serializers/core.py

```
@@ -48,6 +48,24 @@
+    """
+    if value <= 0:
+        raise serializers.ValidationError("Price must be a positive number.")
+
+
+def inventory_is_valid(value: int) -> Union[None, serializers.ValidationError]:
+    """Check if the inventory is a non-negative number.
+    Args:
+        value (int): The inventory quantity of the menu item
+    """
+    if value < 0:
+        raise serializers.ValidationError("Inventory must be a non-negative number.")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)